ivy_tests/test_ivy/test_frontends/test_torch/test_random_sampling.py

```diff
@@ -0,0 +1,41 @@
+# global
+from hypothesis import given
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+@handle_cmd_line_args
+@given(
+    population_size=helpers.ints(),
+    num_samples=helpers.ints(),
+    replace=helpers.ints(),
+    out=helpers.ints(),
```

Comment:
Manage with the `with_out` argument, not manually.

Reply:
I set the with_out argument when I call helpers.test_frontend_function and set with_out = out. I think I still need to initialize out here then? If I just remove this line then how would I initialize with_out?